lib/ProductOpener/Config_opff.pm

```diff
@@ -174,7 +172,31 @@ $flavor = "opff";
 	teolemon
 );
 
-$options{product_type} = "petfood";
+%options = (
+	site_name => "Open Pet Food Facts",
+	product_type => "petfood",
+	og_image_url => "https://world.openpetfoodfacts.org/images/misc/openpetfoodfacts-logo-en.png",
```

Comment:
404

Reply:
https://static.openpetfoodfacts.org/images/misc/openfoodfacts-logo-en-356x300.png ?

Reply:
We don't want the old logo, we will add the new logo later.